process.go

```diff
@@ -6,114 +6,114 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
-func process(src realtimeResponse, serviceName string, dst *prometheusMetrics) {
+func process(src realtimeResponse, serviceName string, serviceID string, dst *prometheusMetrics) {
```

Comment:
Extremely pedantic nit, but can you put serviceID before serviceName in the parameter and label list? :)

Reply:
I'm thinking of continuity of data.  In cases where there are one or more service name changes over time, you either need to | each and every service name alternative over the duration or you get data presentation gaps.  Alternately, if you graph by service IDs then you can freely and regularly rename services to your hearts content.

I'm happy to change the order, I just left it that way because you originally had serviceName as service first ;)